test.sh

```diff
@@ -25,11 +26,12 @@ build_test() {
 
   # hexdump -C test-output-expected/2x2.arrow > test-output-expected/2x2.arrow.hex.txt
   # hexdump -C test-output-actual/2x2.arrow > test-output-actual/2x2.arrow.hex.txt
-
+  if [ "$BASENAME" == "ome-tiff-offsets" ]; then
+    sed -i.bak 's/XMLAnnotation ID=".*"/XMLAnnotation ID=""/g' test-output-actual/ome.xml 
```

Comment:
```suggestion
    sed -i.bak 's/XMLAnnotation ID="[^"]*"/XMLAnnotation ID="PLACEHOLDER"/g' test-output-actual/ome.xml 
```

I think the regexes are greedy by default, so helps to make it tighter to prevent over-matching; And I just like putting an explicit placeholder in there, so if someone is looking at the output without a sense of of it's processed, we can give them something to search on.